src/main/java/org/stellar/sdk/responses/operations/BumpFootprintExpirationOperationResponse.java

```diff
@@ -0,0 +1,24 @@
+package org.stellar.sdk.responses.operations;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Represents BumpFootprintExpiration operation response.
+ *
+ * @see <a href="https://developers.stellar.org/api/horizon/resources/operations"
+ *     target="_blank">Operation documentation</a>
+ * @see org.stellar.sdk.requests.OperationsRequestBuilder
+ * @see org.stellar.sdk.Server#operations()
+ */
+public class BumpFootprintExpirationOperationResponse extends OperationResponse {
+  @SerializedName("ledgers_to_expire")
+  private final Long ledgers_to_expire;
```

Comment:
I don't think snake casing is used as a code style, this should probably be mentioned in #493, using camel case insted such as `ledgersToExpire` is predominant format.

Reply:
could probably make this a lombok @Value class also, and not implement any constructor or getter/setters

Reply:
Sorry, my mistake, fixed in 758746597f480d1df08d55ec0b279dfc61867a89